<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-VAQKFJ2" failures="0" tests="4" name="qa.tests.TC_01_VerifyUserAbleToLoginIntoApplication_DDT" time="10.449" errors="0" timestamp="16 Apr 2024 11:58:42 GMT" skipped="0">
  <testcase classname="qa.tests.TC_01_VerifyUserAbleToLoginIntoApplication_DDT" name="verifyUserAbleToLoginIntoApplication" time="2.904"/>
  <testcase classname="qa.tests.TC_01_VerifyUserAbleToLoginIntoApplication_DDT" name="verifyWhetherApplicationProvidAlertMessageWithWrongUserCredentials" time="2.404"/>
  <testcase classname="qa.tests.TC_01_VerifyUserAbleToLoginIntoApplication_DDT" name="verifyUserAbleToLoginIntoApplication" time="2.119"/>
  <testcase classname="qa.tests.TC_01_VerifyUserAbleToLoginIntoApplication_DDT" name="verifyWhetherApplicationProvidAlertMessageWithWrongUserCredentials" time="3.022"/>
</testsuite> <!-- qa.tests.TC_01_VerifyUserAbleToLoginIntoApplication_DDT -->
